functions/client.ts

```diff
@@ -0,0 +1,20 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+const GRAPHQL_ENDPOINT = 'https://api.uniswap.org/v1/graphql'
+
+//TODO: Figure out how to make ApolloClient global variable
```

Comment:
what does this mean? are you just trying to share a single instance of the client across files, or something else?